admin_manual/installation/source_installation.rst

```diff
@@ -73,6 +72,7 @@ Database connectors (pick the one for your database:)
 * PHP module bz2 (recommended, required for extraction of apps)
 * PHP module intl (increases language translation performance and fixes sorting
   of non-ASCII characters)
+* PHP module sodium (only on RHEL/CentOS to use Argon2 for password hashing. If the password was hashed with Argon2 and the module is missing, your users can't log in.)
```

Comment:
There might be other distros without sodium embedded in PHP core packages, or it can change in the future. I'm suggesting this:
```suggestion
* PHP module sodium (for Argon2 for password hashing. bcrypt is used as fallback, but if passwords were hashed with Argon2 already, and the module is missing, your users can't log in.)
```

Reply:
:+1: